src/app/code/community/Omise/Gateway/controllers/Callback/ValidateoffsitepaynowController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+class Omise_Gateway_Callback_ValidateoffsitepaynowController extends Omise_Gateway_Controller_Base
+{
+    const PAYMENT_TITLE = 'PayNow QR';
```

Comment:
This is appearing in a number of places - sometimes 'PayNow QR', sometimes 'PayNow QR Payment' - should these all be the same? All coming from config?